repos/spack_repo/builtin/packages/netcdf_c/package.py

```diff
@@ -55,14 +55,14 @@ class NetcdfC(CMakePackage, AutotoolsPackage):
     with when("build_system=cmake"):
         # TODO: document why we need to revert https://github.com/Unidata/netcdf-c/pull/1731
         #  with the following patch:
-        patch("4.8.1-win-hdf5-with-zlib.patch", when="@4.8.1: platform=windows")
+        patch("4.8.1-win-hdf5-with-zlib.patch", when="@4.8.1:4.9.2 platform=windows")
 
         # TODO: https://github.com/Unidata/netcdf-c/pull/2595 contains some of the changes
         # made in this patch but is not sufficent to replace the patch. There is currently
         # no upstream PR (or set of PRs) covering all changes in this path.
         # When #2595 lands, this patch should be updated to include only
```

Comment:
I've always found this line's comment confusing -- has this been done? are we still waiting for it? Given the bound on this patch, does that mean that 4.9.3+ includes the full set of changes in this patch?

Reply:
The MPI specific parts of this patch are redundant as of 4.9.3, the hdf5 may still be required but I need to test that further with the MPI behavior and newer versions of hdf5. Since this only effects Windows, lets let this land and I'll figure out what new patch may need to be added. It has since been pointed out to me a few ways this patch can be improved anyway, so I'll likely be upstreaming something that refactors both the MPI and HDF5 handling.